anti-analysis/anti-av/block-operations-on-executable-memory-pages-using-arbitrary-code-guard.yml

```diff
@@ -3,7 +3,7 @@ rule:
     name: block operations on executable memory pages using Arbitrary Code Guard
     namespace: anti-analysis/anti-av
     authors:
-      - jakub.jozwiak@mandiant.com
+      - "jakub.jozwiak@mandiant.com"
```

Comment:
can we do this only when necessary, i.e., when they list github usernames (starting with a special character)?